tests/test-dirs/outline.t/run.t

```diff
@@ -110,21 +110,6 @@
         ],
         "deprecated": false
       },
-      {
-        "start": {
-          "line": 18,
-          "col": 0
-        },
-        "end": {
-          "line": 18,
-          "col": 20
-        },
-        "name": "Ex",
-        "kind": "Exn",
-        "type": null,
-        "children": [],
-        "deprecated": false
-      },
```

Comment:
Looks like this is related to your Type_exception changes, so we can discuss it in the dedicated PR, but are you sure this diff is not a regression ?